### DISCUSSION TITLE
skip or ignore create updated snapshot in specified `snapshot()`

### DISCUSSION THREAD

**Discussion by roccho-dev** [15r10nk/inline-snapshot#46] [Q&A] [answered]
skip or ignore create updated snapshot in specified `snapshot()`

I love this library.
This feature has already been documented?

Thanks.

**Comment by 15r10nk** [comment]
Thank you for your question.

I'm not sure I understand exactly what you want to know.
I have documented the pytest options [here](https://15r10nk.github.io/inline-snapshot/pytest/)

Please let me know if this answers your question.

**Reply by roccho-dev** [reply]
Thanks, excuse my english.

What I want to do are:
- some `snapshot()` in a test file, ex 10 `snapshot()`
- 6 of them are `snapshot()`, which means they are required to be `create`
- I want to `create` 4 of 6
- skip of ignore 2 of 6
- rest of them 4 are `snapshot(xxx)`, which means they are required to be `update`
- and I want to `update` 3 of 4
- skip or ignore 1 of 4

**Reply by 15r10nk** [reply, marked as answer]
No problem, I'm also not a native speaker.

You can use `pytest -k test_function --inline-snapshot=create` to create all snapshots in `def test_function(): ...` (`-k` is a pytest option).

This is also mentioned in the documentation at the end of the Usage section https://15r10nk.github.io/inline-snapshot/#usage.

There is no way to specify which snapshot in a function should be updated.

**little announcement:**
I plan to create a kind of approval-ui for inline-snapshot to provide more control over which snapshots are changed.
This will become the first sponsor only feature for inline-snapshot.
The approval-ui could provide me some income to invest more time in my open source projects. I want to keep core inline-snapshot free of sponsor-only features because I want that every one can use inline-snapshot.
The first version *might* be released this year.

**Reply by roccho-dev** [reply]
Sounds good.
I'm briefly considering becoming a sponsor.
ui means GUI? or API module?

**Reply by 15r10nk** [reply]
It will be a browser based ui, but everything will run locally.
